RecoLocalTracker/SubCollectionProducers/python/SeedClusterRemoverPhase2_cfi.py

```diff
@@ -0,0 +1,10 @@
+import FWCore.ParameterSet.Config as cms
+
+seedClusterRemoverPhase2 = cms.EDProducer("SeedClusterRemoverPhase2",
```

Comment:
cfi file name should start with lower case.
if fillDescriptions was implemented, this file would not be necessary.